test/oli_web/live/sections/edit_live_test.exs

```diff
@@ -196,5 +196,46 @@ defmodule OliWeb.Sections.EditLiveTest do
              |> element("#section_display_curriculum_item_numbering")
              |> render() =~ "checked"
     end
+
+    test "update section with a long title shows an error alert", %{conn: conn, section: section} do
+      {:ok, view, _html} = live(conn, "/authoring/products/#{section.slug}")
```

Comment:
Would be great to not write the explicit path but instead use the route helpers, wdyt?

Reply:
Sure. Perfect! I change it!